src/App.js

```diff
@@ -245,9 +246,29 @@ export const App = ({
     }
   }, [pathname])
 
+  useEffect(() => {
+    const tableRowsCountChangeHandler = ({ detail }) =>
+      setIsLessThan20(detail < 20)
+    window.addEventListener(
+      'tableRowsCountChanged',
+      tableRowsCountChangeHandler,
+      false
+    )
+    return () =>
+      window.removeEventListener(
+        'tableRowsCountChanged',
+        tableRowsCountChangeHandler,
+        false
+      )
+  }, [])
```

Comment:
`[D-0-1]` Instead of propagating the event, we can add the class directly inside the `src/ducks/Watchlists/Widgets/Table/index.js` `useEffect` hook. 
By doing this we will:
1. eliminate additional states in the application root component;
2. encapsulate component's functionality.

Reply:
Viewport class is set in `src/App.js`
If we move this logic to `src/ducks/Watchlists/Widgets/Table/index.js` How we can change App class?

Reply:
`document.querySelector('.App')`